bundles/org.openhab.binding.modbus.sunspec/src/main/resources/ESH-INF/thing/meter-channel-types.xml

```diff
@@ -0,0 +1,125 @@
+		<item-type>Number:Energy</item-type>
+		<label>Total Real Energy Imported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-imported-real-energy-type">
+		<item-type>Number</item-type>
+		<label>Real Energy Imported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="ac-total-exported-apparent-energy-type">
+		<item-type>Number:Energy</item-type>
+		<label>Total Apparent Energy Exported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-exported-apparent-energy-type">
+		<item-type>Number:Energy</item-type>
+		<label>Apparent Energy Exported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-total-imported-apparent-energy-type">
+		<item-type>Number:Energy</item-type>
+		<label>Total Apparent Energy Imported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-imported-apparent-energy-type">
+		<item-type>Number:Energy</item-type>
+		<label>Apparent Energy Imported</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="ac-total-imported-reactive-energy-q1-type">
+		<item-type>Number</item-type>
+		<label>Total Reactive Energy Imported Quadrant 1</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-imported-reactive-energy-q1-type">
+		<item-type>Number:Energy</item-type>
+		<label>Reactive Energy Imported Quadrant 1</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-total-imported-reactive-energy-q2-type">
+		<item-type>Number:Energy</item-type>
+		<label>Total Reactive Energy Imported Quadrant 2</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-imported-reactive-energy-q2-type">
+		<item-type>Number:Energy</item-type>
+		<label>Reactive Energy Imported Quadrant 2</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="ac-total-exported-reactive-energy-q3-type">
+		<item-type>Number:Energy</item-type>
+		<label>Total Reactive Energy Exported Quadrant 3</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-exported-reactive-energy-q3-type">
+		<item-type>Number:Energy</item-type>
+		<label>Reactive Energy Exported Quadrant 3</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-total-exported-reactive-energy-q4-type">
+		<item-type>Number:Energy</item-type>
+		<label>Total Reactive Energy Exported Quadrant 4</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ac-exported-reactive-energy-q4-type">
+		<item-type>Number:Energy</item-type>
+		<label>Reactive Energy Exported Quadrant 4</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+</thing:thing-descriptions>
```

Comment:
Some of our build tools don't like EOF without a newline. So add a newline here.

Reply:
thanks added the newline.